docs/getting-started.md

```diff
@@ -39,6 +30,38 @@ podman run -i --rm quay.io/coreos/fcct:release --pretty --strict < your_config.f
 podman run --rm -v /path/to/your_config.fcc:/config.fcc:z quay.io/coreos/fcct:release --pretty --strict /config.fcc > transpiled_config.ign
 ```
 
+You may also setup the following alias in your shell configuration:
+
+```
+alias fcct='podman run --rm --tty --interactive \
+            --security-opt label=disable        \
+            --volume ${PWD}:/pwd --workdir /pwd \
+            quay.io/coreos/fcct:release'
+```
+
+#### Distribution packages
+
+`fcct` is available from the Fedora packages repositories:
+
+```
+$ sudo dnf install -y fcct
+```
+
+Although this should work, we recommend that you run fcct from the container (as described in the previous section) to make sure that you always use the latest version. If you use the packaged version, make sure that your system is fully updated to the latest Fedora stable release.
+
+#### Standalone binary
+
+Download the latest version of `fcct` and the detached signature from the [releases page](https://github.com/coreos/fcct/releases). Verify it with gpg:
+
+```
+gpg --verify <detached sig> <fcct binary>
+```
+You may need to download the [Fedora signing keys](https://getfedora.org/static/fedora.gpg) and import them with `gpg --import <key>` if you have not already done so.
+
+New releases of `fcct` are backwards compatible with old releases unless otherwise noted.
+
+Although this should work, we recommend that you run fcct from the container (as described in the first section) to make sure that you always use the latest version.
```

Comment:
Using the standalone binary is somewhat worse than the distro package, but I'd still be inclined to drop this line.